main.tf

```diff
@@ -136,6 +155,7 @@ resource "restapi_object" "cspm" {
       target_accounts = var.cspm_enabled ? [
         {
           account_id         = ibm_resource_instance.scc_wp.account_id
+          account_type       = module.account_type_check[0].account_type
```

Comment:
i think this will cause an issue when cspm is disabled because you have a count around `module.account_type_check` however we always run this restapi patch.